stdlib/public/Darwin/Accelerate/vForce_Operations.swift

```diff
@@ -0,0 +1,1781 @@
+                          result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Double, V.Element == Double {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvtan(dest.baseAddress!,
+                          src.baseAddress!,
+                          &n)
+                }
+            }
+    }
+    
+    /// Calculates the tangent of pi multiplied by each element in `vector`, writing the result to `result`, single-precision.
+    ///
+    /// - Parameter vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func tanPi<U, V>(_ vector: U,
+                            result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Float, V.Element == Float {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvtanpif(dest.baseAddress!,
+                             src.baseAddress!,
+                             &n)
+                }
+            }
+    }
+    
+    /// Calculates the tangent of pi multiplied by each element in `vector`, writing the result to `result`, double-precision.
+    ///
+    /// - Parameter vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func tanPi<U, V>(_ vector: U,
+                            result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Double, V.Element == Double {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
+            
+            result.withUnsafeMutableBufferPointer { dest in
+                vector.withUnsafeBufferPointer { src in
+                    vvtanpi(dest.baseAddress!,
+                            src.baseAddress!,
+                            &n)
+                }
+            }
+    }
+    
+    // MARK: Arcsine
+    
+    /// Calculates the arcsine of each element in `vector`, writing the result to `result`, single-precision.
+    ///
+    /// - Parameter vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func arcsin<U, V>(_ vector: U,
```

Comment:
This should match whatever spelling the math functions proposal settles on (currently `asin`).

Reply:
Changed (also, `acos` and `tan`). Obviously I can change if the proposal settles on something else.